docs/rules/jsx-no-bind.md

```diff
@@ -62,6 +65,26 @@ When `true` the following is **not** considered a warning:
 <div onClick={this._handleClick.bind(this)} />
 ```
 
+### `customMessage`
+
+When `customMessage` is set, it will override the default error message for each rule violation,
+for example, with the following config:
+
+```js
+"react/jsx-no-bind": [true, {
+  "customMessage": "A bind call in a JSX prop will cause performance issues related to garbage collection"
```

Comment:
Let's just call this "message", but let's also improve the default such that as few people as possible would need/want to customize it.